api/base/views.py

```diff
@@ -1,30 +1,31 @@
 import weakref
+
 from django.conf import settings as django_settings
 from django.http import JsonResponse
-from rest_framework.decorators import api_view
-from rest_framework.response import Response
+
 from rest_framework import generics
-from rest_framework import status
 from rest_framework import permissions as drf_permissions
+from rest_framework import status
+from rest_framework.decorators import api_view
 from rest_framework.exceptions import ValidationError, NotFound
+from rest_framework.mixins import ListModelMixin
+from rest_framework.response import Response
 
 from framework.auth.oauth_scopes import CoreScopes
 
-from rest_framework.mixins import ListModelMixin
 from api.base import permissions as base_permissions
+from api.base import utils
 from api.base.exceptions import RelationshipPostMakesNoChanges
 from api.base.filters import ListFilterMixin
-
-from api.users.serializers import UserSerializer
-from api.base.parsers import JSONAPIRelationshipParser
-from api.base.parsers import JSONAPIRelationshipParserForRegularJSON
+from api.base.parsers import JSONAPIRelationshipParser, JSONAPIRelationshipParserForRegularJSON
```

Comment:
So the advantage of having two lines for this import is that, if you add or delete a new item, there is only a single line change in the comparison, rather than an add and delete. Also, be careful about doing minor changes like this for stylistic reasons; this is the sort of thing that causes merge conflicts (if someone adds one parser and someone else removes another parser, that would cause trouble).